setup.cfg

```diff
@@ -0,0 +1,96 @@
+[metadata]
+author = SciTools Developers
+author_email = scitools-iris-dev@googlegroups.com
+classifiers =
+    Development Status :: 5 Production/Stable
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Operating System :: MacOS
+    Operating System :: POSIX
+    Operating System :: POSIX :: Linux
+    Operating System :: Unix
+    Programming Language :: Python
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Atmospheric Science
+    Topic :: Scientific/Engineering :: Visualization
+description =
+    A powerful, format-agnostic, community-driven Python package for analysing and visualising Earth science data
+download_url = https://github.com/SciTools/iris
+install_requires =
+    cartopy>=0.18
+    cf-units>=2
+    cftime<1.3.0
+    dask[array]>=2
+    matplotlib
+    netcdf4
+    numpy>=1.14
+    scipy
+    scitools-pyke
+    xxhash
+keywords =
+    cf-metadata
+    data-analysis
+    earth-science
+    grib
+    netcdf
+    meteorology
+    oceanography
+    space-weather
+    ugrid
+    visualisation
+license = LGPL-3.0-or-later
+license_file = COPYING.LESSER
+long_description = file: README.md
+long_description_content_type = text/markdown
+name = scitools-iris
+project_urls =
+    Code = https://github.com/SciTools/iris
+    Discussions = https://github.com/SciTools/iris/discussions
+    Documentation = https://scitools-iris.readthedocs.io/en/stable/
+    Issues = https://github.com/SciTools/iris/issues
+url = https://github.com/SciTools/iris
+version = attr: iris.__version__
+
+[options]
+include_package_data = True
+packages = find:
+package_dir =
+    =lib
+python_requires =
+    >=3.6
```

Comment:
```suggestion
    >=3.7
```

#4163

Reply:
To be fair #4163 hadn't been merge :stuck_out_tongue_winking_eye: 

I'll rebase this PR now, to pull in those changes and tweak appropriately :+1:

Reply:
I know, I'm not trying to make you look bad!

Reply:
Done :+1: